### ISSUE TITLE
How to get a string of numbers starting with 0 from the JSON

### ISSUE THREAD

**Issue by sr5220** [microsoft/ClearScript#262]
For example:
`{
  "05220": "hello world"
}`
I can get the V8ScriptItem
but "05220" became 5220

**Comment by ClearScriptLib** [comment]
Hi @sr5220,

This is a bug. We'll have a fix in the next release. Thanks for reporting it!

BTW, if you can share the code with which you discovered this bug, please do.

Thanks again!

**Comment by sr5220** [comment]
Thanks for your quick response!

Code:
```C#
using (var engine = new V8ScriptEngine())
{
    engine.Execute("var testJson = { \"05220a\": \"hellow world\", \"05220\": \"hellow world\" }");
}
```

I can get the value by "05220".but "PropertyIndices" shows 5220.

**Comment by ClearScriptLib** [comment]
Fixed in [Version 7.1.3](https://github.com/microsoft/ClearScript/releases/tag/7.1.3).
